soc/xtensa/intel_adsp/ace/ace-link.ld

```diff
@@ -266,9 +266,11 @@ SECTIONS {
     *(.gnu.version_r)
     KEEP (*(.eh_frame))
     KEEP (*crtbegin.o(.ctors))
+#ifndef CONFIG_CPP_STATIC_INIT_GNU
```

Comment:
How about this one-line comment? `rodata section for xtensa overrides the .ctors section`.

Reply:
@marc-hb Please re-review this PR.

Reply:
> @marc-hb Please re-review this PR.

OK, unlike the earlier versions now I can easily understand the (brand new) code, thank you. However I still have no idea whether this is the Right Way to do this or not, sorry. @tejlmand, @stephanosio  can you help?